profile/benchmark_helper.rb

```diff
@@ -0,0 +1,185 @@
+require 'mongo'
+require 'json'
+require 'FileUtils'
+
+class BenchmarkHelper
+  # Accessor methods for
+  attr_reader :database, :collection
+
+
+  # Initializes a client connection, creating a database and a collection.
+  #
+  # @example Initialize a client connection.
+  #   BenchmarkHelper.initialize_collection("testing")
+  #
+  # @param [ String, Symbol ] collection_name The name of the collection.
+  # @param [ String, Symbol ] database_name The name of the database.
+  #
+  # @return [ Mongo::Collection ] The collection.
+  #
+  # @since 2.2.1
+  def initialize(database_name, collection_name, pool_size = 5)
+    Mongo::Logger.level = Logger::INFO
+    @client = Mongo::Client.new(
+        [ '127.0.0.1:27017' ],
+        database: database_name,
+        write: { :w => 1 },
+        max_pool_size: pool_size
+    )
+    @database = @client.database
+    @collection = @client[collection_name]
+  end
+
+
+  class << self
+
+
+    # Load a file into a string
+    #
+    # @example Load a file into a string
+    #   BenchmarkHelper.load_string_from_file("GRIDFS_LARGE.txt")
+    #
+    # @param [ String ] data_file_name The name of the data file.
+    #
+    # @return [ String ] A string of all the file data.
+    #
+    # @since 2.2.1
+    def load_string_from_file(data_file_name)
+      file = File.open(data_file_name, "rb")
```

Comment:
is there a reason you passed File.open a block in load_array_from_file and not here? I would go with the block approach.